src/pages/kb/user-guide/querying/scheduling-a-query.md

```diff
@@ -6,23 +6,28 @@ title: Scheduling A Query
 slug: scheduling-a-query
 ---
 
-By default, your query won't have any scheduling but it's super easy to
-adjust!
-
-In the bottom left corner, you'll see the schedule area:
+You can use scheduled query executions to keep your public dashboards updated or to power routine Alerts. By default, your queries will not have a schedule. But this is easy to adjust. In the bottom left corner of the query editor you'll see the schedule area:
 
 <img src="/assets/images/docs/gitbook/query-details.png" width="100%">
 
-Clicking ' Never ' will open a picker for "every x" or at a specific time of
-day. Note: The time you set the scheduling of queries is based on the timezone
-of the computer you're using to set it (then it's translated to UTC).
+Clicking `Never` will open a picker with allowed schedule intervals. Note: The time you set the scheduling of queries is based on the timezone of the computer you're using to set it (then it's translated to UTC).
```

Comment:
Is it clear that we store the UTC value?

Reply:
Fixed with an explainer c43fcf0